src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Tov.cpp

```diff
@@ -27,20 +27,19 @@
 /// \cond
 namespace {
 
-void lindblom_rhs(
-    gsl::not_null<std::array<double, 2>*> dvars,
-    const std::array<double, 2>& vars, const double log_enthalpy,
-    const std::unique_ptr<EquationsOfState::EquationOfState<true, 1>>&
-        equation_of_state) noexcept {
+void lindblom_rhs(gsl::not_null<std::array<double, 2>*> dvars,
```

Comment:
could you please factor the `unique_ptr` -> reference change into a separate commit?